src/Distributions.jl

```diff
@@ -4,7 +4,7 @@ using StatsBase, PDMats, StatsFuns, Statistics
 using StatsFuns: logtwo, invsqrt2, invsqrt2π
 
 import QuadGK: quadgk
-import Base: size, length, convert, show, getindex, rand, vec, inv
+import Base: in, size, length, convert, show, getindex, rand, vec, inv
```

Comment:
Can you instead qualify `in` in the definition of `in(::Real, ::RealInterval)`? IMO we should try to avoid generic `import`s to make it clearer when and which functions are overloaded.

Reply:
Should be better now